### PR TITLE
Add dot-env-secrets recipe

### PR DESCRIPTION
**Pull request by lucianosrp**
## Description

What's cooking? 
As discussed with @EmilRex, here is my first recipe to help loading enviroment variables from a .env file to a Prefect workspace using the Secret Block. Not sure where this should be in the repository.  

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## New Recipe Checklist
- [x] My PR is in the format of `Add <project-name> recipe`
- [x] My recipe is reproducible and explains everything needed to run successfully. If my code has external dependencies, I make mention of them.
- [x] My code is easily understandable and/or well-commented.
- [ ] If my recipe requires a new category (e.g. creating a monitoring/ folder in devops/), I have encapsulated my project within its own subfolder so that others can add their own recipes as well.
- [ ] If my recipe uses Prefect < 2.0, my code is within the prefect-v1-legacy/ folder.

### CODE REVIEW

File: flows-advanced/dot-env-secrets/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Dot-env Secrets
+
+Recipe to quickly load your `.env` secrets into
+Prefect's [Secret block](https://docs.prefect.io/latest/api-ref/prefect/blocks/system/#prefect.blocks.system.Secret).
+
+## Reasoning
+
+Often times, I would find myself adapting existing Python script into Prefect.
```
Comment (zzstoatzz) [inline flows-advanced/dot-env-secrets/README.md:8]:
```suggestion
Oftentimes, I find myself adapting existing Python scripts into Prefect.
```

**Review by zzstoatzz — COMMENTED** [review]
hi @lucianosrp - thank you for the contribution 😄 

I think this is a cool example, I'm not sure this (`load_secrets`) needs to be a flow though

would you be open creating a new folder at the same level as `flows-advanced` called `configuration` or something? we were thinking of `flows-advanced` as a place for more advanced flow authoring strategies

**Comment by lucianosrp** [comment]
Hi @zzstoatzz , I agree! I really didn't know if it had to be a flow in order to be in this repository. 
I like your idea and I will make this change! Thanks for having a look at it 🙏

**Review by zzstoatzz — APPROVED** [review]
thank you so much for the contribution @lucianosrp !

**Pull request merged**